src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -592,12 +572,13 @@ private StudentEnrollDetails enrollStudent(StudentAttributes validStudentAttribu
     }
 
     /* All empty lines or lines with only white spaces will be skipped.
-     * The invalidity info returned are in HTML format.
+     * The invalidity info is in HTML format.
```

Comment:
As we are changing the method behavior, we can also fix this header comment. It should start with `/**`.  Describe when does it throws the exception.